lib/watch-files.js

```diff
@@ -64,7 +63,9 @@ module.exports = (nameOfDirectory, interval = 5000) => {
         logResult('change', `occurred on ${file}, queueing upload`, 'magentaBright');
       }
       queue.push(file);
-      queueData.set(file, { pattern, type });
+      if (!queueData.has(file)) {
+        queueData.set(file, { pattern, type });
```

Comment:
file is like `experience/components/errorAlert.hbs` so file will be unique, pattern and type will never change so I'm just keeping them in the map.